src/db/sysdb_search.c

```diff
@@ -511,6 +510,41 @@ errno_t sysdb_search_ts_matches(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    ret = ldb_search(sysdb->ldb, tmp_ctx, &res, NULL,
+                     LDB_SCOPE_SUBTREE, attrs, "%s", filter);
+    if (ret) {
+        ret = sysdb_error_to_errno(ret);
+        goto done;
+    }
+
+    *_res = talloc_steal(mem_ctx, res);
+    ret = EOK;
+
+done:
+    talloc_zfree(tmp_ctx);
+    return ret;
+}
+
+static errno_t sysdb_enum_dn_filter(TALLOC_CTX *mem_ctx,
+                                    struct ldb_result *ts_res,
+                                    const char *filter,
```

Comment:
I know the original code you refactored was also guilty of this, but could you please name this variable "name_filter" same as it was named in the caller? Because the only attribute we use this as a value of is SYSDB_NAME, the variable name might as well make it clear..